.github/CODEOWNERS

```
@@ -1038,9 +1038,23 @@ x-pack/test_serverless/api_integration/test_suites/common/platform_security @ela
 /x-pack/plugins/entity_manager @elastic/obs-entities
 /x-pack/test/api_integration/apis/entity_manager @elastic/obs-entities
 
+
 # Data Discovery
-/test/plugin_functional/plugins/data_search @elastic/kibana-data-discovery
+/test/functional/fixtures/es_archiver/alias @elastic/kibana-data-discovery
+/test/functional/page_objects/context_page.ts @elastic/kibana-data-discovery
+/test/functional/services/data_views.ts @elastic/kibana-data-discovery
+/test/functional/services/saved_objects_finder.ts @elastic/kibana-data-discovery
+/test/functional/page_objects/unified_search_page.ts @elastic/kibana-data-discovery
 /test/plugin_functional/plugins/index_patterns @elastic/kibana-data-discovery
+/test/plugin_functional/plugins/data_search @elastic/kibana-data-discovery
+/test/functional/page_objects/discover_page.ts @elastic/kibana-data-discovery
+/test/functional/fixtures/es_archiver/index_pattern_without_timefield @elastic/kibana-data-discovery
+/test/functional/fixtures/es_archiver/huge_fields @elastic/kibana-data-discovery
+/test/functional/fixtures/es_archiver/date_n* @elastic/kibana-data-discovery
+/test/functional/firefox/discover.config.ts @elastic/kibana-data-discovery
+/test/functional/fixtures/es_archiver/discover/**/*.ts @elastic/kibana-data-discovery
+/test/api_integration/apis/suggestions @elastic/kibana-data-discovery
```

```suggestion
/test/api_integration/apis/suggestions @elastic/kibana-visualizations
```

As of now UnifiedSearch is owned by the Visualizations team. Same for `/test/functional/page_objects/unified_search_page.ts`.

@davismcphee Or would you be fine with assigning it to us?

---

I'd suggest assigning it to @elastic/kibana-visualizations for now to keep things simpler. If it exchanges ownership in the future, we can reassign the test files too.

---

https://github.com/elastic/kibana/pull/200132/commits/85292c5d9e7a483c33a12067d73a9f658499992d